source/pages/Settings/ImportAccount.tsx

```diff
@@ -0,0 +1,188 @@
+import { Menu, Transition } from '@headlessui/react';
+import { Input, Form } from 'antd';
+import { useForm } from 'antd/es/form/Form';
+import React, { useEffect, Fragment, useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import { Layout, Icon, DefaultModal, NeutralButton } from 'components/index';
+import { useUtils } from 'hooks/index';
+import { RootState } from 'state/store';
+import { getController } from 'utils/browser';
+import { validatePrivateKeyValue } from 'utils/validatePrivateKey';
+
+const ImportAccountView = () => {
+  const controller = getController();
+  const { navigate, alert } = useUtils();
+  const [form] = useForm();
+  const { importAccountFromPrivateKey } = controller.wallet;
+
+  //* States
+  const [type, setType] = useState('Private Key');
+  const [isAccountImported, setIsAccountImported] = useState(false);
+  const [isImporting, setIsImporting] = useState(false);
+
+  const { accounts, activeAccount: activeAccountId } = useSelector(
+    (state: RootState) => state.vault
+  );
+
+  const activeAccount = accounts[activeAccountId];
+
+  const isUnlocked =
+    controller.wallet.isUnlocked() && activeAccount.address !== '';
+
+  if (!activeAccount) throw new Error('No account');
```

Comment:
do we need a new Error in here? This sounds odd to me, since it is a component, the normal react behavior is just don't render the component, something like `return null`